examples/hashing/Makefile

```diff
@@ -0,0 +1,12 @@
+APPLICATION = hashing
```

Comment:
This should rather be a test that an example.

Reply:
I've moved it to tests/hashing.